src/PantryLink/PantryLink.react.js

```diff
@@ -0,0 +1,124 @@
+import classnames from 'classnames';
```

Comment:
i did this since we have Link/LinkBase wrappers in ATK. so Link in atk would have to render Link in pantry. the names conflicted. maybe i should be changing the names of the wrappers in ATK instead?